### PR TITLE
Add charter for GC subgroup

### PR DESCRIPTION
**Pull request by rossberg**
C+P courtesy of the [WASI charter](https://github.com/WebAssembly/WASI/blob/master/Charter.md).

### CODE REVIEW

**Comment by lars-t-hansen** [comment]
Looks reasonable enough.

File: proposals/gc/Charter.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- new specifications in standards bodies or working groups
+  (e.g. W3C WebAssembly WG or Ecma TC39),
+
+- new specifications outside of standards bodies
+  (e.g. similar to the LLVM object file format documentation in Wasm tool conventions).
+
+### Non-normative reports
+
+The Subgroup may produce non-normative material such as requirements
+documents, recommendations, and use cases.
```
Comment (RossTate) [inline proposals/gc/Charter.md]:
```suggestion
documents, recommendations, and case studies.
```

Just want to emphasize a larger-scale analysis of how languages will compile to WebAssembly. Use cases tend to focus on features one at a time, whereas case studies further address the interaction of features.

**Comment by RossTate** [comment]
Modulo the one suggestion, looks good! Thanks for putting this together.

**Comment by kripken** [comment]
Looks good to me too!

**Comment by rossberg** [comment]
Okay, thanks everybody. As a formality, I defer landing this until we have CG approval to form the subgroup.

**Pull request merged**